iree/vm/bytecode_module_impl.h

```diff
@@ -16,6 +16,13 @@
 #define IREE_VM_BYTECODE_MODULE_IMPL_H_
 
 #include <stdint.h>
+#include <string.h>
+
+// TODO(benvanik): figure out how to make MSVC happy with C11 stdalign.h.
```

Comment:
FYI: This will also have problem with Android, where we don't have `cstdalign`. `stdalign.h` is fine, at least with ndk-r21d.

Reply:
https://github.com/google/iree/commit/5589fd2c6f83fbea62dd885c57fad1b04ec40948 is a possible fix to make it compile. Ben might want to see whether it makes sense though.